docs/development/features.md

```diff
@@ -90,6 +94,12 @@ Note:
 
 :heavy_check_mark: Signed builds
 
+:heavy_check_mark: Registry Credential Helper
```

Comment:
Isn't this mostly the same thing as "Registry Configuration" above under "Image Management"?

Reply:
Registry Credential Helper is the feature we just released in 1.4.1.. 

"Registry Configuration" is the feature we are targeting as part of the configuration/settings GUI in an upcoming release. May be name it "Private Registry Configuration" would make it a bit more clear?

Reply:
> "Registry Configuration" is the feature we are targeting as part of the configuration/settings GUI in an upcoming release. May be name it "Private Registry Configuration" would make it a bit more clear?

I don't know; what kind of GUI do you expect? There is really nothing to configure providing access credentials, which is done either via credential helpers, or via client certificates, which are stored under `~/.docker/certs.d`.

Maybe we want to allow configuring helpers via the GUI, but that doesn't feel like a pressing need.

The one thing currently not easily configurable are "insecure registries", i.e. registries that either don't support https (or that use a self-signed cert, and the user doesn't want to install the corresponding CA).

Reply:
I was under the impression there's more functionality to "Registry Management" and we are planning to have a GUI screen to capture credentials etc.. I will remove the "Registry Credential Helper" item.